src/domain/enrolment/enrolmentForm/EnrolmentForm.tsx

```diff
@@ -74,6 +76,32 @@ export const defaultInitialValues: EnrolmentFormFields = {
   },
 };
 
```

Comment:
This is quite long file now due to these type definitions, I would rather move them to separate file.

Reply:
moved to contants.ts